### ISSUE TITLE
welcome page accessibility and formatting need improvement

### ISSUE THREAD

**Issue by AlanFCMV** [ucfopen/UDOIT#652]
Currently the welcome page heading levels. It skips directly from h2 for "welcome to UDOIT" to h4 for the individual issue names inside the expandable cards. 

There are several elements on the page that should probably be a heading as well, such as the "please note:" and "other resources" text, the "What does UDOIT look for?" button, or the Errors and Suggestions under that button.

It is awkward that the name for each individual issue appears twice, once on the button and one on the h4 for the expandable card. Perhaps those buttons should also be headings.

Additionally, the alt text for the image that serves as an h1 is "UDOIT logo." This should probably be changed to just "UDOIT."

**Comment by cidilabs** [comment]
I agree that we need to have order in our headings, but I would suggest less headings over adding more headings. The issue name repeated is unnecessary, so let's just remove the heading within the card. Less is more. :)

**Comment by bagofarms** [comment]
I think that the expandable accordion names should be H3s, but things like "Please note:" and "other resources" are fine to leave as normal text.
